backend/dataall/modules/datasets/cdk/dataset_data_policy.py

```diff
@@ -0,0 +1,33 @@
+from typing import List
+from aws_cdk import aws_iam as iam
+
+from dataall.cdkproxy.stacks.policies.data_policy import DataPolicy
+from dataall.modules.datasets.db.models import Dataset
+from dataall.modules.datasets.services.dataset_service import DatasetService
+
+
+class DatasetDataPolicy(DataPolicy):
```

Comment:
Maybe we can rename it S3 policy as it is what is actually doing and then everything is a service policy

Reply:
Right now "data" can be anything